pkg/reconciler/autoscaling/kpa/resources/decider.go

```diff
@@ -65,6 +65,7 @@ func MakeDecider(ctx context.Context, pa *v1alpha1.PodAutoscaler, config *autosc
 			MaxScaleUpRate:      config.MaxScaleUpRate,
 			ScalingMetric:       pa.Metric(),
 			TargetValue:         target,
+			TargetRPS:           config.RPSTargetDefault,
```

Comment:
As @yanweiguo mentioned, this shouldn't be needed.

Reply:
Still here.